labelbox/schema/project.py

```diff
@@ -371,6 +375,71 @@ def _validate_datetime(string_date: str) -> bool:
                          self.uid)
             time.sleep(sleep_time)
 
+    """
+    Creates a project run export task with the given params and returns the task.
+    
+    >>>    export_task = export_v2("my_export_task", filter={"media_attributes": True})
+    
+    """
+
+    def export_v2(self, task_name: str,
+                  params: Optional[ProjectExportParams]) -> Task:
+        defaultParams: ProjectExportParams = {
+            "attachments": False,
+            "media_attributes": False,
```

Comment:
This always is requested. So let's remove it from the params.

Reply:
I'll fix in a follow up PR